extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -37,6 +37,7 @@ const APP = '__AMPHTML__';
  */
 const RequestNames = {
   CHANNEL_OPEN: 'channelOpen',
+  DISABLE_SCROLL: 'disableScroll',
```

Comment:
@chenshay @peterjosling Wasn't there a `scrollLock` message already? Is this the same thing or something different entirely?

Reply:
`scrollLock` is handled in `touch-handler.js` and disables touch-based scrolling only by calling `preventDefault()` on any `touchmove` events that fire. That won't stop a user from scrolling with a keyboard/mouse, and will block any scrolling on the page including our use case where we want the parent page to continue to scroll.